tools/bridge/bridge/confirmation_sender.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict
+
+import gevent
+from gevent.queue import Queue
+
+from web3.contract import Contract
+from eth_keys.datatypes import PrivateKey
+from eth_utils import keccak, int_to_big_endian
+
+from bridge.constants import HOME_CHAIN_STEP_INTERVAL
```

Comment:
```suggestion
from bridge.constants import HOME_CHAIN_STEP_DURATION
```
Nitpicking: I think if this relates to the chain specification it should have the same postfix. Moreover I think step interval doesn't even make sense. At least for this context. :man_shrugging: